src/charm.py

```diff
@@ -38,6 +39,10 @@
 
 logger = logging.getLogger(__name__)
 
+ROOT_PASSWORD_KEY = "root-password"
+SERVER_CONFIG_PASSWORD_KEY = "server-config-password"
+CLUSTER_ADMIN_PASSWORD_KEY = "cluster-admin-password"
```

Comment:
would these be better placed in `constants.py`?

Reply:
Thanks Shayan! Done.